package/tests/test_server.py

```diff
@@ -197,6 +197,19 @@ def test_no_browser(cli_runner):
     assert server.webbrowser.open_new.called
```

Comment:
I know it's not part of this PR, but lines 175 and 184 use `.called_with` method instead of [`assert_called_with`](https://docs.python.org/3.6/library/unittest.mock.html#unittest.mock.Mock.assert_called_with). I think `called_with` is not a standard mock method, so it will return just another mock. If this is true, then this is a bug.

Could you please try changing it to `assert_called_with`?

Reply:
good spot, fixed